### DISCUSSION TITLE
Action always fail with `HttpError`

### DISCUSSION THREAD

**Discussion by ix64** [ArtiomTr/jest-coverage-report-action#206] [Q&A] [answered]
Hello, I am new to `jest-coverage-report-action`, I am using the most simple configuration.
But it always fail with `HttpError` even I can receive the coverage here 
https://github.com/unlock-music/unlock-music/commit/bd66016cb59b135a00cb4ec4d7c04b86133f7af9#commitcomment-62089919
Did I make a mistake somewhere? I would be grateful if I could get your help!

## Logs
https://github.com/unlock-music/unlock-music/runs/4569718652
```log
Begin failed tests' annotations publication...
Failed tests' annotations publication skipped
Failed tests' annotations publication ended
Begin coverage annotations publication...
Coverage annotations publication failed
Error: HttpError: Invalid request.

"end_line", "start_line" weren't supplied.
Coverage annotations publication ended
Error: Jest coverage report action failed
Error: Process completed with exit code 1.
```

## Workflow 
https://github.com/unlock-music/unlock-music/blob/master/.github/workflows/build.yml#L25-L32
```yaml
  test-coverage:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - run: npm ci
      - uses: ArtiomTr/jest-coverage-report-action@v2.0-rc.6
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by ArtiomTr** [comment, marked as answer]
Hey @ix64 👋,

This bug is already registered #188. All further information will be published there.
